### PR TITLE
Renaming file Get-CsBatchTeamsDeployment.md to Get-CsBatchTeamsDeploymentStatus

### PR DESCRIPTION
**Pull request by srinidhikMS**

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws @get-itips @tseward

File: teams/teams-ps/teams/Get-CsBatchTeamsDeploymentStatus.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
 external help file: Microsoft.TeamsCmdlets.PowerShell.Custom.dll-Help.xml
 Module Name: MicrosoftTeams
-online version: https://docs.microsoft.com/powershell/module/teams/get-csbatchteamsdeployment
+online version: https://docs.microsoft.com/powershell/module/teams/Get-CsBatchTeamsDeploymentStatus
 schema: 2.0.0
 ---
 
-# Get-CsBatchTeamsDeployment
+# GGet-CsBatchTeamsDeploymentStatus
```
Comment (dariomws) [inline teams/teams-ps/teams/Get-CsBatchTeamsDeploymentStatus.md:8]:
```suggestion
# Get-CsBatchTeamsDeploymentStatus

```

**Review by dariomws — CHANGES_REQUESTED** [review]
In the Description section the cmdlet name must also be updated.

**Review by get-itips — APPROVED** [review]
I am ok after @dariomws suggestions applied

**Review by dariomws — APPROVED** [review]

**Comment by srinidhikMS** [comment]
> In the Description section the cmdlet name must also be updated.

Cmdlet name is updated in description. Thanks.

**Comment by yogkumgit** [comment]
@tseward Gentle reminder please.

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**